src/main/java/com/facebook/openwifirrm/modules/ApiServer.java

```diff
@@ -424,6 +431,90 @@ private String getOpenApiJson(Request request, Response response) {
 		return Json.pretty(getOpenApi());
 	}
 
+	// TODO [ZoneBasedRrmScheduling] remove this once we move to venues
+	/**
+	 * Calls prov to get a mapping of all RRM enabled devices to their settings
+	 * and updates the device config.
+	 */
+	private void getAndUpdateRRMDetails() {
```

Comment:
As discussed, when Prov sync is enabled then this isn't needed as we should be blocking topology changes, as well as config changes that specifically modify Prov-managed RRM settings (this is much more annoying to check). You can add some TODOs to the config setters like the one in `/setToplogy` and get to this later.